frontend/src/pages/modelServing/screens/projects/nimUtils.ts

```diff
@@ -123,3 +124,59 @@ export const updateServingRuntimeTemplate = (
   }
   return updatedServingRuntime;
 };
+
+export const getNIMResourcesToDelete = async (
+  projectName: string,
+  servingRuntime: ServingRuntimeKind,
+): Promise<Promise<void>[]> => {
+  const resourcesToDelete: Promise<void>[] = [];
+  try {
+    const count = await fetchInferenceServiceCount(projectName);
+    if (count === 0) {
+      return resourcesToDelete;
+    }
```

Comment:
Since you updated the logic to throw an error -- instead of returning 0 -- the only way to get into this block is if you don't have the resources... did we want to try and delete the PVC still? Or... not?

Reply:
the if (count === 0) condition has been removed.